src/main/scala/firrtl_interpreter/ExternalModule.scala

```diff
@@ -37,9 +37,9 @@ case class BlackBoxOutput(name: String,
   def mapType(f: Type => Type): Expression = this
   def mapWidth(f: Width => Width): Expression = this
   def foreachExpr(f: Expression => Unit): Unit = f(this)
-  def foreachType(f: Type => Unit): Unit = Unit
-  def foreachWidth(f: Width => Unit): Unit = Unit
-  def execute(inputValues: Seq[Concrete]): Concrete = {
+  def foreachType(f: Type => Unit): Unit = ()
+  def foreachWidth(f: Width => Unit): Unit = ()
+  def execute(inputValues: immutable.Seq[Concrete]): Concrete = {
```

Comment:
Rather than specifying this is `immutable.Seq`, I'd remove the `import scala.collection._` that can have lots of unintended consequences, eg. `dependentInputs` above is now `scala.collection.Seq` when that's probably not what we want in 2.13

Reply:
Reworked this. looks much cleaner